### DISCUSSION TITLE
Lifetime of containers in Integration Tests

### DISCUSSION THREAD

**Discussion by megashrieks** [microsoft/aspire#6877] [General]
Hi, I see that containers with lifetime as persistent as Lifetime are persisting even after the integration tests are run.

Is this the correct behavior?

**Comment by anhaehne** [comment]
I thinks this can be converted into an issue. Even if this behaviour is expected, the containers all get unique names so they aren't reused anyways. 

I would like this to be configurable. Test isolation is important but some containers are just to slow to start to not reuse them.

**Reply by anhaehne** [reply]
I just saw there is already an open issue that was elevated to a bug: #6850
